### ISSUE TITLE
Is it possible to access the subset of RadImageNet used in the pre-trained models?

### ISSUE THREAD

**Issue by adolfogc** [BMEII-AI/RadImageNet#1]
Hi,

Is it possible to access the subset of the RadImageNet dataset that was used for producing the published pre-trained weights?

I see that the models were trained using 3-channel images but medical images from, say MRI or CT, are greyscale. Based on the images available in this repo, I'm guessing that the 3-channel images were produced by stacking the same greyscale data, is this correct?

It would be interesting to compare a model pre-trained on the greyscale dataset with a greyscale version of the ImageNet subset, as [available in Kaggle](https://www.kaggle.com/c/imagenet-object-localization-challenge/overview/description), similarly to as done by [1], which report improved results over training in the RGB version in a transfer learning application of chest X-ray image classification. Another interesting possibility is trying this dataset for unsupervised feature learning, for example PixPro [2], applied to the segmentation of medical images.

Regarding the pre-processing used in the examples:

https://github.com/BMEII-AI/RadImageNet/blob/1e098bd4ce9539bbeb56fad5b07367cbd050449b/acl/acl_train.py#L153

I see that the pre-processing function for ImageNet is used, but as mentioned in Tensorflow's documentation, it:

> will convert the images from RGB to BGR, then will zero-center each color channel with respect to the ImageNet dataset, without scaling

Do you have the mean and standard deviation learned from the RadImageNet subset? So as to use those for the normalization instead of using the parameters learned from ImageNet?

Finally, what is the license of the published models/weights? For example, would it be possible to use them in a Research Code Competition from Kaggle?

Thanks in advance.

--

[1]  Xie, Y., & Richmond, D. (2018). Pre-training on grayscale imagenet improves medical image classification. In Proceedings of the European Conference on Computer Vision (ECCV) Workshops (pp. 0-0).

[2] Xie, Z., Lin, Y., Zhang, Z., Cao, Y., Lin, S., & Hu, H. (2021). Propagate yourself: Exploring pixel-level consistency for unsupervised visual representation learning. In Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition (pp. 16684-16693).

**Comment by adolfogc** [comment]
Hi, thank you for your reply.
